examples/lpr/lpr.py

```diff
@@ -0,0 +1,262 @@
+"""
+LPR analysis for amorphous silicon dataset
+==========================================
+
+.. start-body
```

Comment:
This is not needed! 

Can you add the authors for this example, like we do for the other ones here https://github.com/lab-cosmo/software-cookbook/pull/46?